### ISSUE TITLE
bin/isofit still tries to initialize Isofit instance with row_column argument, which is no longer supported

### ISSUE THREAD

**Issue by NickVaughn** [isofit/isofit#297]
Trying to run examples/20171108_Pasadena/run_example_modtran.sh ...
```
Traceback (most recent call last):
File "<isofit repo>/bin/isofit", line 7, in <module>
  exec(compile(f.read(), __file__, 'exec'))
File "<isofit repo>bin/isofit", line 54, in <module>
  main()
File "<isofit repo>/bin/isofit", line 47, in main
  isofit = Isofit(config_file=config, level=level, row_column=row_column)
TypeError: Isofit.__init__() got an unexpected keyword argument 'row_column'
```

**Comment by pgbrodrick** [comment]
Thanks Nick!  Currently being addressed in #313 (partially complete).

**Comment by jammont** [comment]
Resolved by PR #313
